brainreg_napari/register.py

```diff
@@ -176,6 +204,9 @@ def widget(
         registration_output_folder: pathlib.Path
             Where to save the registration output
                 affine_n_steps: int,
+        save_original_orientation: bool
+            Option to save annotations with the same orientation as the input data. Use this if you plan to map
+            segmented objects outside brainglobe/tools.
```

Comment:
I know it is like this in the rest of the docstring, but I see no benefit of having the typing in the docstring and in the function definition. If, for whatever reason, the function definition changes, there is a risk that the docstring gets out of sync.

Reply:
This is useful for type checking and hints in IDE.

Reply:
Thanks @JulesScholler I knew there was a reason why I did this, it's for the IDE!

Reply:
I don't know what IDE you're using, but for me just having in function definition is enough 🤷

Reply:
Pycharm. I think sphinx is also parsing the doc string and not the function argument to generate the documentation so that's another benefit of adding the typing in the docstring.